docs/YIP/YIP-0005-session-key-management.md

```diff
@@ -0,0 +1,57 @@
+# 0005 - Session Key Management for ClearSync
+
+## Status
+
+Work in progress
+
+## Context
+
+The adoption of session keys enhances security and operational efficiency.
+By limiting the validity of keys and associating them with specific counterparts, we significantly reduce risks associated with key exposure and unauthorized access.
+
+This proposal introduces a new strategy for key management in the ClearSync project, shifting from signing sub-keys to registering session keys.
+Each Clearport will generate a key pair limited to a default duration of 24 hours, termed a clearing session.
+
+## Decision
+
+### Key Generation and Registration
+
+- **Generation**: Each Clearport generates a key pair valid for a default duration of 24 hours.
+- **Registration**: Participants must register the clearing public key on the adjudicator, specifying the duration, counterpart, and owner (Smart-Account).
```

Comment:
Is the session key generated and registered on the adjudicator at the moment of connection?

Reply:
If you want to start the channel you need to register and init your session key because it will require on-chain transaction so we need to be careful here to do not to create a session key without further usage and useless transaction from the user